docs/topics/pages.md

```diff
@@ -381,6 +409,7 @@ content_panels = [
 ```
 
 The first argument must match the value of the `related_name` attribute of the `ParentalKey`.
+For a brief description of parameters taken by `InlinePanel`, see [Inline Panel](https://docs.wagtail.org/en/latest/reference/pages/panels.html#inlinepanel) 
```

Comment:
Avoid using full URLs when linking between documentation pages, as the docs are likely to be re-hosted in various different locations (for example, a user might be viewing a specific pinned version of the docs rather than `/latest/`).
```suggestion
For a brief description of parameters taken by `InlinePanel`, see [Inline Panel](inline_panels).
```

For this link to work, you'll need to reinstate the `(inline_panels)=` anchor at the top of the new expanded InlinePanel section in reference/pages/panels.